couchdbkit/client.py

```diff
@@ -437,7 +437,9 @@ def list(self, list_name, view_name, **params):
         vname = '/'.join(list_name)
         list_path = '_design/%s/_list/%s/%s' % (dname, vname, view_name)
 
-        return self.res.get(list_path, **params).json_body
+        res = self._request_session.get(self._database_path(list_path), params=params)
+        res.raise_for_status()
```

Comment:
Guessing this is not tested, but wondering if this will raise the same errors that would have been raised previously? Would be good to preserve the interface by catching and re-throwing errors with whatever exception type was used previously (if different).

Same for other uses of `raise_for_status()`